src/services/api.js

```diff
@@ -185,7 +185,10 @@ export class YdbEmbeddedAPI extends AxiosWrapper {
                 timeout: 600000,
```

Comment:
Is it a backend timeout? Why 2 timeouts?

Reply:
one for backend operations, one for the client request